examples/fabric-admin/commands/clusters/ModelCommand.cpp

```diff
@@ -35,7 +35,7 @@ CHIP_ERROR ModelCommand::RunCommand()
         return SendGroupCommand(GroupIdFromNodeId(mDestinationId), fabricIndex);
     }
 
-    ChipLogProgress(chipTool, "Sending command to node 0x%" PRIx64, mDestinationId);
+    ChipLogProgress(chipTool, "Sending command to node 0x%lx", mDestinationId);
```

Comment:
The latest CI complain the old way, I think this check was added recently.

Reply:
Is CI complaining about `PRIx64` or, `ChipLogFormatX64`, with `ChipLogValueX64`?

Reply:
It complains " 0x%" PRIx64, after I switched to the traditional way like '0x%lx', the CI passed

Reply:
Okay this goes back to my original question:

Is there a reason you are using `%lx` instead of `ChipLogFormatX64`, with `ChipLogValueX64`?

I only ask because all the other times I have seen NodeId printed I have seen `ChipLogFormatX64`, with `ChipLogValueX64` used

Reply:
No specific reason to pass the CI, updated to use ChipLogFormatX64 to see if it can clear the CI.